FPGA1394_QLA/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ set (VERILOG_SOURCE_COMMON "${VERILOG_SOURCE_DIR}/ClkDiv.v"
                            "${VERILOG_SOURCE_DIR}/CtrlAdc.v"
                            "${VERILOG_SOURCE_DIR}/BoardRegs.v"
                            "${VERILOG_SOURCE_DIR}/DS2505.v"
+                           "${VERILOG_SOURCE_DIR}/uart_recv_2480.v"
```

Comment:
I see there is also a uart_send_2480.v.  Should that be added to CMakeLists.txt?